open-api.yaml

```diff
@@ -0,0 +1,269 @@
+openapi: 3.0.3
+info:
+  title: ATLAS feeding_data
+  description: Send planned rations and get actual feeding reports into the system.
+  version: '0.0.2'
+servers:
+  - url: http://localhost:8080
+    description: Inferred Url
+security: 
+  - "bearerAuth": []
+paths:
+  /feedingreport:
+    get:
+      summary: feedingReport
+      parameters:
+        - name: start_date
+          in: query
+          description: Start date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: end_date
+          in: query
+          description: End date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: ration_ref
+          in: query
+          description: ration_ref represents identification for connecting the rations between systems
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RecipeDailyReport'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+  /ration:
+    post:
+      summary: importRation
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/RationRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+        '201':

```

Comment:
Either 204 or 201 but both?

Reply:
I replaced 200 with 204 response and removed 201